TeamCode/src/main/java/org/firstinspires/ftc/teamcode/ultimategoal/modules/OdometryModule.java

```diff
@@ -34,7 +39,10 @@
     public double rightPodNewPosition;
     public double mecanumPodNewPosition;
 
+    private long startTime = Calendar.getInstance().getTime().getTime();
```

Comment:
??? What happened to SystemClock.getCurrentTime

Reply:
devs nerfed it